client/components/password-change/index.js

```diff
@@ -0,0 +1,15 @@
+import {connect} from "react-redux";
+
+import Component from "./password-change";
+
+const mapStateToProps = state => {
+  return {
+    passwordConfirm: state.organization.configuration.components.confirm_form,
```

Comment:
@someshkar , `organization.configuration.components.confirm_form` variable you are using is declared [here](https://github.com/openwisp/openwisp-wifi-login-pages/blob/299460ed0b73cb9bc03821afed51667561861425/org-configurations/default-configuration.yml#L138). Check how settings for the password reset confirm form is defined inside that variable. Similary you have to declare a variable `password_change_from` and define all the settings for the password change component inside it. In this way, we can keep the settings configurable at organization level.